spotifycli/spotifycli.py

```diff
@@ -98,106 +129,190 @@ def get_arguments():
         ("--playpause", "plays or pauses the song (toggles a state)"),
         ("--lyrics", "shows the lyrics for the song"),
         ("--next", "plays the next song"),
-        ("--prev", "plays the previous song")
+        ("--prev", "plays the previous song"),
+        ("--export", "export lyrics"),
     ]
 
 
+def set_genius_token(token):
+    os.environ[TOKEN_ENV_VAR] = token
+
+    with open(os.path.join(expanduser("~"), TOKEN_FILE_NAME), "w") as f:
```

Comment:
I think that polluting `$HOME` directory with additional temporary file may not be the best idea. I personally don't like when random programs create random files in my `$HOME` dir without my knowledge.